Jenkinsfile.cloud

```diff
@@ -253,6 +253,7 @@ node(NODE) {
     def ami_intermediate = utils.sh_capture("jq -r .HVM ${dirpath}/aws-${AWS_REGION}-smoketested.json")
     build job: 'coreos-rhcos-aws-test', wait: false, parameters: [
         string(name: 'ami_intermediate', value: "${ami_intermediate}"),
+        string(name: 'version', value: "${version}"),
```

Comment:
It looks like these parameters aren't defined in the AWS job's properties.  Is the server logging security warnings about them?

Reply:
Offhand I don't see any, though I don't know the specific place to look for that.

Reply:
On the Container Linux Jenkins masters, they'd be in the `jenkins.service` journal output.

Reply:
I don't see any warnings of that form.

Reply:
It looks like we're running with `-Dhudson.model.ParametersAction.keepUndefinedParameters=true`, which is... not good. We'd have to ask the other tenants of this Jenkins why that is; I distinctly remember having to deal with this in the past. Though I'd agree it'd be better to explicitly add them as parameters.

Reply:
So, I can turn this on in the next master restart. It looks like this is the only job that would need fixing, so we can adapt it first and then turn it on.